config/src/lib.rs

```diff
@@ -187,6 +187,12 @@ pub struct Config {
     pub path_pattern_inverse: Option<globset::Glob>,
     /// The number of test cases that must execute for each property test
     pub fuzz_runs: u32,
+    /// The number of calls executed to attempt to break invariants in one run.
+    pub invariant_depth: u32,
```

Comment:
I think a separate `invariant_runs` parameter (analogous to `fuzz_runs`) would be a great addition, because 1 invariant run is more time consuming than 1 fuzz run